tests/sagemaker/mock/__init__.py

```diff
@@ -1076,10 +1075,7 @@ def response_object(self):
         return response
 
 
-# Create a SageMaker backend for each EC2 region
-sagemaker_backends = {}
-for region, ec2_backend in ec2_backends.items():
-    new_backend = SageMakerBackend()
-    sagemaker_backends[region] = new_backend
+# Create a SageMaker backend for EC2 region: "us-west-2"
+sagemaker_backends = {"us-west-2": SageMakerBackend()}
```

Comment:
According to the description of https://github.com/spulec/moto/pull/4699:
> all backends are now generated on-request, rather then on start-up.

So we need to create a SageMaker backend for "us-west-2", which is used in our test. Since "us-west-2" is **hardcoded** in our tests as the default AWS region, I also hardcoded it here.

With moto==2.3.0, the old implementation only returns one region "us-east-1", which is the default region used by moto.

Reply:
We can probably update the small-requirements.txt now to instead of excluding version 2.0.7, just force use of >=2.3.0.

Reply:
This change is also compatible with moto<2.3.0, so we don't have to change it now.